src/mol-io/reader/sdf/parser.ts

```diff
@@ -95,10 +97,11 @@ function handleMolFile(tokenizer: Tokenizer) {
 
     const atoms = molIsV3 ? handleAtomsV3(tokenizer, atomCount) : handleAtoms(tokenizer, atomCount);
     const bonds = molIsV3 ? handleBondsV3(tokenizer, bondCount) : handleBonds(tokenizer, bondCount);
+    const formalCharges = handleFormalCharges(tokenizer,1)
```

Comment:
Please format the documents (e.g. missing comma and ``;`` here) (``npm run lint-fix`` should do the trick).